faiss/python/setup.py

```diff
@@ -16,17 +16,29 @@
 shutil.copyfile("__init__.py", "faiss/__init__.py")
 shutil.copyfile("loader.py", "faiss/loader.py")
 shutil.copyfile("swigfaiss.py", "faiss/swigfaiss.py")
+if os.path.exists("swigfaiss_avx2.py"):
+    shutil.copyfile("swigfaiss_avx2.py", "faiss/swigfaiss_avx2.py")
 
 ext = ".pyd" if platform.system() == 'Windows' else ".so"
 prefix = "Release/" * (platform.system() == 'Windows')
-shutil.copyfile(f"{prefix}_swigfaiss{ext}", f"faiss/_swigfaiss{ext}")
 
-try:
-    shutil.copyfile("swigfaiss_avx2.py", "faiss/swigfaiss_avx2.py")
-    shutil.copyfile(f"{prefix}_swigfaiss_avx2{ext}", f"faiss/_swigfaiss_avx2{ext}")
-except Exception as e:
-    print(f"Could not move swigfaiss_avx2.py / {prefix}_swigfaiss_avx2{ext} due to:\n{e!r}")
-    pass
+swigfaiss_general_lib = f"{prefix}_swigfaiss{ext}"
+swigfaiss_avx2_lib = f"{prefix}_swigfaiss_avx2{ext}"
+
+found_swigfaiss_general = os.path.exists(swigfaiss_general_lib)
+found_swigfaiss_avx2 = os.path.exists(swigfaiss_avx2_lib)
+
+assert (found_swigfaiss_general or found_swigfaiss_avx2), \
+        f"Could not find {swigfaiss_general_lib} or " \
+        f"{swigfaiss_avx2_lib}. Faiss may not be compiled yet."
```

Comment:
```suggestion
if not (found_swigfaiss_general or found_swigfaiss_avx2):
    raise FileNotFoundError(f"Could find neither {swigfaiss_general_lib} nor "
                            f"{swigfaiss_avx2_lib}. Faiss may not be compiled yet.")
```

Reply:
Your suggestion is reasonable but I think a simple assertion is enough here.

Reply:
Ultimately up to the maintainers here, but I think bare asserts (outside of tests) are bad style. You should return a specific error.